app/(afterLogin)/dashboard/_components/course-permitted/index.tsx

```
@@ -1,132 +1,83 @@
+                                {item.description}
+                              </div>
+                              <div className="text-gray-500 text-xs">
+                                {item.date}
+                              </div>
+                            </div>
+                            <div className="text-sm p-3 bg-primary rounded-lg text-white">
+                              {item.team}
+                            </div>
+                          </div>
+                        </Card>
+                      ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)